exporter/collectors/network.py

```diff
@@ -2,30 +2,50 @@
 
 class NetworkCollector:
 
+    
     def __init__(self) -> None:
         """
         TODO: @
         Constructor
         """
-        pass
+        
+    def get_unit(self, bytes):
+        units = ['B', 'KB', 'MB', 'GB', 'TB']
+        size = bytes
+        for unit in units:
+            if size < 1024:
+                return f"{size:.1f}{unit}"
+        size /= 1024
 
     def get_traffic_in(self):
         """
         TODO: @
         Returns the inbound traffic in Mb/s.
         """
-        pass
+        netStat = psutil.net_io_counters()
+        return self.get_unit(netStat.bytes_recv) 
 
     def get_traffic_out(self):
         """
         TODO: @
         Returns the outbound traffic in Mb/s.
         """
-        pass
+        netStat = psutil.net_io_counters()     
+        return self.get_unit(netStat.bytes_sent) 
+
+
+
 
     def __str__(self) -> str:
         """
         TODO:
         Returns to string representation of all networking metrics.        
         """
-        pass
+       
+        traffic_in = self.get_traffic_in()
+        traffic_out = self.get_traffic_out()
+        return f"Inbound Traffic: {traffic_in} Mb/s, Outbound Traffic: {traffic_out} Mb/s"
+
+network_collector = NetworkCollector()
```

Comment:
**Requested Changes**
It seems you've left some of your code from testing in this file. The collector files should not print anything out themselves. Code in Question:

> network_collector = NetworkCollector()
> print(network_collector.__str__())